week-1/errors/4.js

```diff
@@ -1,2 +1,7 @@
 const 12HourClockTime = "20:53";
-const 24hourClockTime = "08:53";
\ No newline at end of file
+const 24hourClockTime = "08:53";
+// variables must start with letters
+
+const twelveHourClockTime = "20:53";
+const twentyFourhourClockTime = "08:53";
```

Comment:
You may want to consider swapping the times.
"20:53" looks like twentyFourhourClockTime format 😉